Extension/c_cpp_properties.schema.json

```diff
@@ -36,10 +36,14 @@
                             "c99",
                             "c11",
                             "c17",
+                            "c20",
```

Comment:
I'm not sure if c20 is going to be a thing.  There was ambiguity between c17 and c18 at one point as well and we settled on just keeping one of them.  I "think" we just need c23 in the schema.

Reply:
There is no c20 defined for gcc https://gcc.gnu.org/onlinedocs/gcc/C-Dialect-Options.html#C-Dialect-Options  and presumably clang as well, as it copies it's std's from gcc https://clang.llvm.org/docs/UsersManual.html  ("The supported modes for C are c89, gnu89, c94, c99, gnu99, c11, gnu11, c17, gnu17, c2x, gnu2x, and various aliases for those modes.")

Reply:
Where's the source of truth?

Reply:
@michelleangela it was a placeholder based off a reference PR

@Colengms so c2x is the only argument we track, right?

Reply:
@Bernardin-MS I'm not sure I understand your question.  "c23" and "gnu23" are valid.  But, "c20" and "gnu20" aren't valid, so could you remove those?  :)